src/flownet/config_parser/_config_parser_pred.py

```diff
@@ -70,6 +90,64 @@ def create_schema(_to_abs_path) -> Dict:
                             "max_running": {MK.Type: types.Integer},
                         },
                     },
+                    "ref_sim": {
+                        MK.Type: types.String,
+                        MK.Transformation: _to_abs_path,
+                        MK.Description: "Reference simulation to be used in analysis",
```

Comment:
Including the extension? (`.DATA`?)

Reply:
Same as for input_case in config for ahm. Works both with and without .DATA extension.